### PR TITLE
Change addresses for tokens in Gnosis Chain and Mainnet

### PR DESCRIPTION
**Pull request by anxolin**
Adds addresses for xDAI and Mainnet for USDC, WETH/wXDAI and GNO

### Test Plan

Please, verify the addresses point to the right token

### CODE REVIEW

File: src/tasks/test-deployment.ts [hunk 1]
```diff
@@ -31,12 +31,19 @@ const OUTPUT_FOLDER = "./output";
 
 const defaultTokens = {
   usdc: {
+    "1": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+    "100": "0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83",
     "4": "0x4DBCdF9B62e891a7cec5A2568C3F4FAF9E8Abe2b",
   },
   weth: {
```
Comment (josojo) [inline src/tasks/test-deployment.ts:38]: @fedgiac 
here, we could have used the name nativeToken. Maybe we can adapt it in a follow up PR

**Review by josojo — APPROVED** [review]
I checked the addresses. Looks good

File: src/tasks/test-deployment.ts (hunk 1 above)
Comment (fedgiac) [reply src/tasks/test-deployment.ts:38]: Agreed. :+1:

**Review by fedgiac — APPROVED** [review]
Looks good!

**Review by fedgiac — APPROVED** [review]
Thanks for the PR, feel free to merge to main.

**Pull request merged**